src/target/target.c

```diff
@@ -4781,18 +4890,29 @@ static int target_jim_get_reg(Jim_Interp *interp, int argc,
 
 COMMAND_HANDLER(handle_set_reg_command)
 {
-	if (CMD_ARGC != 1)
+	if (CMD_ARGC < 1 || CMD_ARGC > 2)
+		return ERROR_COMMAND_SYNTAX_ERROR;
+
+	bool flush = false;
+	const char *force_arg = NULL;
+
+	if (CMD_ARGC == 3)
```

Comment:
There seems to be a typo.

Note that in upstream I have already corrected this. (https://review.openocd.org/c/openocd/+/8070)

```suggestion
	if (CMD_ARGC == 2)
```

Reply:
You are right, thanks!

Reply:
Addressed (Updated the cherry-pick to the latest patch https://review.openocd.org/c/openocd/+/8070/21)